internal/namespaces/object/v1/custom_bucket.go

```diff
@@ -0,0 +1,559 @@
+package object
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-cli/v2/internal/human"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+type bucketInfo struct {
+	ID               string
+	Region           scw.Region
+	APIEndpoint      string
+	BucketEndpoint   string
+	EnableVersioning bool
+	Tags             []types.Tag
+	ACL              []CustomS3ACLGrant
+	Owner            string
+}
+
+func bucketInfoMarshalerFunc(i interface{}, opt *human.MarshalOpt) (string, error) {
+	// To avoid recursion of human.Marshal we create a dummy type
+	type tmp bucketInfo
+	info := tmp(i.(bucketInfo))
+
+	opt.Sections = []*human.MarshalSection{
+		{
+			FieldName:   "Tags",
+			HideIfEmpty: true,
+		},
+		{
+			FieldName:   "ACL",
+			HideIfEmpty: true,
+		},
+	}
+	str, err := human.Marshal(info, opt)
+	if err != nil {
+		return "", err
+	}
+	return str, nil
+}
+
+type BucketResponse struct {
+	SuccessResult *core.SuccessResult
+	BucketInfo    bucketInfo
+}
+
+func bucketResponseMarshalerFunc(i interface{}, opt *human.MarshalOpt) (string, error) {
+	resp := i.(BucketResponse)
+
+	messageStr, err := resp.SuccessResult.MarshalHuman()
+	if err != nil {
+		return "", err
+	}
+	bucketStr, err := bucketInfoMarshalerFunc(resp.BucketInfo, opt)
+	if err != nil {
+		return "", err
+	}
+
+	return strings.Join([]string{
+		messageStr,
+		bucketStr,
+	}, "\n"), nil
+}
+
+type bucketConfigArgs struct {
+	Region           scw.Region
+	Name             string
+	Tags             []string
+	EnableVersioning bool `json:"enable-versioning"`
+	ACL              string
+}
+
+func bucketCreateCommand() *core.Command {
+	return &core.Command{
+		Namespace: "object",
+		Resource:  "bucket",
+		Verb:      "create",
+		Short:     "Create an S3 bucket",
+		Long:      "Create an Object Storage Bucket with the S3 protocol. The namespace is shared between all S3 users, so its name must be unique.",
+		ArgsType:  reflect.TypeOf(bucketConfigArgs{}),
+		ArgSpecs: core.ArgSpecs{
+			{
+				Name:       "name",
+				Positional: true,
+				Required:   true,
+				Short:      "The unique name of the bucket",
+			},
+			{
+				Name:       "tags",
```

Comment:
List have to be specified with index
```suggestion
				Name:       "tags.{index}",
```